tests/test_mechaclient.py

```diff
@@ -0,0 +1,168 @@
+"""
+test_mechaclient.py - tests for src.mechaclient class
+
+Copyright (c) 2019 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE
+"""
+import pytest
+
+from src.mechaclient import MechaClient
+from src.packages.fact_manager import FactManager
+from src.packages.board.rat_board import RatBoard
+from src.packages.cache.rat_cache import RatCache
+
+from src.packages.commands import command
+from src.packages.context.context import Context
+
+pytestmark = pytest.mark.mechaclient
+
+
+def test_verify_version(bot_fx):
+    """
+    Asserts the version is set, and returning string.
+    """
+    assert bot_fx.__version__
+    assert isinstance(bot_fx.__version__, str)
+
+
+def test_verify_api_handler_private(bot_fx):
+    """
+    Asserts the private reference for the API handlers is none, after instantiation.
+    """
+    assert bot_fx._api_handler is None
+
+
+def test_verify_fact_manager_private(bot_fx):
+    """
+    Asserts the private reference for the Fact Manager is none, after instantiation.
+    """
+    assert bot_fx._fact_manager is None
+
+
+def test_verify_rat_cache_private(bot_fx):
+    """
+    Asserts the private reference for the Rat Cache is none, after instantiation.
+    """
+    assert bot_fx._rat_cache is None
+
+
+def test_verify_rat_board_private(bot_fx):
+    """
+    Asserts the private reference for the Rat Board is none, after instantiation.
+    """
+    assert bot_fx._rat_board is None
+
+
+def test_verify_api_handler(bot_fx):
+    """
+    Asserts the api_handler property returns an object, and cannot be set (no setter)
```

Comment:
[error] this is not what the test is doing.

Reply:
Docstring fixed in 96dac7b.